src/exodus/wallet.cpp

```diff
@@ -169,13 +226,24 @@ void Wallet::OnMintAdded(
     const SigmaPublicKey& pubKey,
     int block)
 {
+    LOCK(pwalletMain->cs_wallet);
+
     SigmaMintId id(property, denomination, pubKey);
+    auto isMintTracked = HasSigmaMint(id);
 
-    if (!HasSigmaMint(id)) {
-        return;
-    }
+    if (isMintTracked) {
```

Comment:
`isMintTracked` is not quite correct due to we don't have tracker anymore. Just use `HasSigmaMint(id)` directly will be better.